build_che.sh

```diff
@@ -16,7 +16,9 @@ scl enable rh-maven33 'mvn -B clean install'
 cd ..
 
 git clone -b ${GIT_BRANCH} ${GIT_REPO}
-cd che 
+cd che
+# Use our modified stacks
+mv ./ide/che-core-ide-stacks/src/main/resources/stacks.json.centos \ 
+ ./ide/che-core-ide-stacks/src/main/resources/stacks.json
```

Comment:
@l0rd but it means that if I work with che I will need to revert this change manually. One more point - Should not we build `che-core-ide-stacks` component in the script ?